EngineLayer/Proteomics/PeptideWithSetModifications.cs

```diff
@@ -59,6 +61,14 @@ public PeptideWithSetModifications(PeptideWithSetModifications modsFromThisOne,
             this.numFixedMods = numFixedMods;
             this.allModsOneIsNterminus = allModsOneIsNterminus ?? new Dictionary<int, ModificationWithMass>();
         }
+        public PeptideWithSetModifications(Protein protein, DigestionParams digestionParams, int oneBasedStartResidueInProtein, int oneBasedEndResidueInProtein, string peptideDescription, int missedCleavages,
```

Comment:
this is fine, but try removing the other constructor and see what breaks and talk to me about it